target/src/package.json

```diff
@@ -1,10 +1,9 @@
 {
   "name": "antlr4ts",
-  "version": "0.4.0-dev",
+  "version": "0.4.1-alpha.0",
```

Comment:
OK, I'm going to punt this PR for now as I think the mechanisms of Lerna may be superior.